features/checkout/paying_for_order/inform_customer_about_order_total_changes.feature

```diff
@@ -12,7 +12,7 @@ Feature: Inform customer about any order total changes during checkout process
         And the store ships everywhere for Free
         And the store allows paying Offline
 
-    @ui
+    @ui @no-api
```

Comment:
I wonder if it's not a valid API scenario as well. (besides the "I shouldn't see a thank you page" stop).
I've recreated this scenario in the API, and IMO our current ordering behavior can be confusing for the customer as we can complete the order even if the price has been changed, and the customer can find out at the payment gateway or when the courier arrives

Reply:
`I should be informed that order total has been changed` is a flash message checker and thank you page doesn't exist. It's a frontend team duty to handle such cases.

Reply:
This is a very similar case to the promotion eligibility checker [scenarios](https://github.com/Sylius/Sylius/blob/1.13/features/checkout/order_promotion/order_promotion_integrity_validation.feature#L24), this check takes place after the order has been processed in the backend, so we should also handle it in the API 🖖🏻

Reply:
I checked it, and I guess we should discuss it on a daily. I still believe we shouldn't move this behavior 1:1 to API.